numba/tests/test_parfors.py

```diff
@@ -1956,6 +1956,47 @@ def test_impl(x, sz):
             return x
         self.check(test_impl, np.zeros((10, 10)), 3)
 
+    def test_untraced_value_tuple(self):
+        # This is a test for issue #6478.
+        def pyfunc():
+            a = (1.2, 1.3)
+            return a[0]
+
+        cfunc = jit(nopython=True, parallel=True)(pyfunc)
+        self.assertEqual(cfunc(), pyfunc())
+
+    def test_recursive_untraced_value_tuple(self):
+        # This is a test for issue #6478.
+        def pyfunc():
+            a = ((1.2, 1.3),)
+            return a[0][0]
+
+        cfunc = jit(nopython=True, parallel=True)(pyfunc)
+        self.assertEqual(cfunc(), pyfunc())
```

Comment:
Same comment as above.  Replace with builtin mechanism for running both and testing results with self.check.